decidim-core/app/views/decidim/participatory_processes/show.html.erb

```diff
@@ -18,3 +18,5 @@
     </div>
   </div>
 <% end %>
+
+<%= comments_for participatory_process %>
```

Comment:
As second thoughts maybe you are right about not including comments at all. I think about two solutions here:

- As you said in another comment, create a DummyComponent only present in `decidim-dev- and test against it.
- Block this PR for now and continue working using this as base merge.